README.md

```diff
@@ -1,5 +1,8 @@
 # Urban-Heat-Island
 Studying urban heat island effect in Austin, Texas using Google Earth Engine.
 
+## Running GOOGLE Earth Engine
+Using google earth engine requires an gmail, install ```geemap``` then, run this command in your virtual environment ```earthengine authenticate```. It will require you to follow the authentication process.
```

Comment:
I will try this tomorrow ...

Reply:
I couldn't find a way to share an api key, but if you have a gmail I think I can add you as a user for the project.

Reply:
OK cool thanks, you can add bobbelderbos@gmail.com

Reply:
Any more progress, questions, things I can help you with?

Reply:
yes, but its now in main.py because I changed the name of the file. hello.py -> main.py

Reply:
Took some time to understand the roles and permission but I add you to some. Let me know if you're able to make it work whenever you have time. Thanks